### PR TITLE
Add changes for edge-20.3.2

### PR DESCRIPTION
**Pull request by zaharidichev**
## edge-20.3.2

This release introduces substantial proxy improvements as well as
new observability and security functionality.

* CLI
  * Added the `linkerd alpha stat` command, which uses the smi-metrics
    API; the latter enables access to metrics to be controlled with RBAC 
* Controller
  * Improved the `destinations` service to allow it to set the TLS identity
    based on metadata present on the `Endpoints` objects of services
  * Added support for configuring service profile timeouts
    `(x-linkerd-timeout)` via OpenAPI spec (thanks @ lewiscowper!)
* Web UI
  * Improved the Grafana dashboards to use a globing operator for Prometheus in
    order to avoid producing queries that are too large (thanks @mmiller1!)
* Helm
  * Improved the `linkerd2` chart README (thanks @lundbird!)
* Proxy
  * Fixed a bug that could cause log levels to be processed incorrectly
  * Removed cache capacity limitations, allowing services to fail eagerly
  * Improved backpressure support

Signed-off-by: Zahari Dichev <zaharidichev@gmail.com>

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+
+This release introduces substantial proxy improvements as well as
+new observability and security functionality.
+
+* CLI
+  * Added the `linkerd alpha stat` command, which uses the smi-metrics
+    API; the latter enables access to metrics to be controlled with RBAC 
+* Controller
+  * Improved the `destinations` service to allow it to set the TLS identity
+    based on metadata present on the `Endpoints` objects of services
```
Comment (alpeb) [inline CHANGES.md]: Maybe not enough user-facing to be included?

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+  * Added support for configuring service profile timeouts
+    `(x-linkerd-timeout)` via OpenAPI spec (thanks @ lewiscowper!)
```
Comment (alpeb) [inline CHANGES.md]: The space in `@ lewiscowper` forbids it to become a link

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
+* Web UI
+  * Improved the Grafana dashboards to use a globing operator for Prometheus in
+    order to avoid producing queries that are too large (thanks @mmiller1!)
+* Helm
+  * Improved the `linkerd2` chart README (thanks @lundbird!)
+* Proxy
+  * Fixed a bug that could cause log levels to be processed incorrectly
+  * Removed cache capacity limitations, allowing services to fail eagerly
```
Comment (adleong) [inline CHANGES.md]: Is there a user-facing consequence of this?

File: CHANGES.md [hunk 4]
```diff
@@ -1,3 +1,26 @@
...
+  * Improved backpressure support
```
Comment (adleong) [inline CHANGES.md]: Is there a user-facing consequence of this?

**Pull request merged**